pydocx/export/image_resize.py

```diff
@@ -0,0 +1,165 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+
+import base64
+from StringIO import StringIO
+from urlparse import urlparse
+import posixpath
+
+import requests
+
+from requests.exceptions import InvalidSchema
+
+from pydocx.util import uri
+from PIL import Image
```

Comment:
Yeah, I agree. I wasn't sure how you guys wanted this kept separate from the main code. Thought I'd send up the pull-request as-is for us to discuss. Happy to move things around to make it all optional.

However, as I was working on this, I struggled to think of a time when you **wouldn't** want the large images resized. Since images are being converted to DataURIs, it seems rather silly to have large images that are scaled down be kept at full size.

Can you help me think of some scenarios where you wouldn't want the behavior?